### PR TITLE
feat(config): add 'contentful/rich-text' monorepo

### PR DESCRIPTION
**Pull request by a-ide**
## Changes:

Adds contentful/rich-text monorepo to config preset.

## Context:

https://github.com/contentful/rich-text

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ const repoGroups = {
   'azure azure-storage-net': 'https://github.com/Azure/azure-storage-net',
   'bugsnag-js': 'https://github.com/bugsnag/bugsnag-js',
   'chakra-ui': 'https://github.com/chakra-ui/chakra-ui',
+  'contentful-rich-text': 'https://github.com/contentful/rich-text',
```
Comment (JamieMagee) [inline lib/config/presets/internal/monorepo.ts:28]: You can also use `contentful rich-text` if you'd like. I don't have a preference either way, and can merge this as-is if you'd prefer 👍

**Review by JamieMagee — APPROVED** [review]

File: lib/config/presets/internal/monorepo.ts (hunk 1 above)
Comment (viceice) [reply lib/config/presets/internal/monorepo.ts:28]: @JamieMagee no whitespace and only lowercase. We have and issue for fixing older groups.

**Review by viceice — APPROVED** [review]

File: lib/config/presets/internal/monorepo.ts (hunk 1 above)
Comment (JamieMagee) [reply lib/config/presets/internal/monorepo.ts:28]: TIL. Thanks for preventing me from making that mistake!

Comment (viceice) [reply lib/config/presets/internal/monorepo.ts:28]: fore reference #10210 🙃

**Pull request merged**